analysis_options.yaml

```diff
@@ -7,7 +7,7 @@
 
 # The following line activates a set of recommended lints for Flutter apps,
 # packages, and plugins designed to encourage good coding practices.
-include: package:flutter_lints/flutter.yaml
+# include: package:flutter_lints/flutter.yaml
```

Comment:
Lets not disable this. In the future our CI will enforce linting.

Thoughts @sanjh